### PR TITLE
Updating with two new activities for URL Links

### PR DESCRIPTION
**Pull request by cebulnes**

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks

File: Teams/audit-log-events.md [hunk 1]
```diff
@@ -151,6 +151,9 @@ Here's a list of all events that are logged for user and admin activities in Tea
 |Updated tab   |TabUpdated         |A user modified a tab in a channel.         |
 |Upgraded app |AppUpgraded           |An app was upgraded to its latest version in the catalog.     |
 |User signed in to Teams     |TeamsSessionStarted         |A user signs in to a Microsoft Teams client. This event doesn't capture token refresh activities.         |
+|User sent a message with a URL link in Teams     |MessageCreatedHasLink         |A user sends a message containing a URL Link in Teams.         |
```
Comment (JohanFreelancer9) [inline Teams/audit-log-events.md:154]:
```suggestion
|User sent a message with a URL link in Teams     |MessageCreatedHasLink         |A user sends a message containing a URL link in Teams.         |
```

File: Teams/audit-log-events.md [hunk 2]
```diff
@@ -151,6 +151,9 @@ Here's a list of all events that are logged for user and admin activities in Tea
...
+|User edits a message with a URL link in Teams     |MessageEditedHasLink         |A user edits a message and adds a URL Link to it in Teams.         |
```
Comment (JohanFreelancer9) [inline Teams/audit-log-events.md:155]:
```suggestion
|Edited a message with a URL link in Teams     |MessageEditedHasLink         |A user edits a message and adds a URL link to it in Teams.         |
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
@cebulnes - Please commit my minor suggestions to align with the style of the table.

**Comment by cebulnes** [comment]
@JohanFreelancer9  done! Thanks

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by scanum** [comment]
@markjjo PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Comment by markjjo** [comment]
@cebulnes PR merged. I re-ordered as activities in the table are listed in alphabetical order

**Comment by cebulnes** [comment]
Thanks @markjjo do we know when this will be live?

**Comment by markjjo** [comment]
@cebulnes This should be live by the end of the day today.

**Comment by markjjo** [comment]
@cebulnes FYI that this PR is live: https://docs.microsoft.com/en-us/microsoftteams/audit-log-events#teams-activities

**Pull request merged**